### PR TITLE
Update myrackspace-username-updates.md

### PR DESCRIPTION
**Pull request by butlercp**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-83147c51a5/how-to/)

File: content/dedicated-hosting/myrackspace-username-updates.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 permalink: myrackspace-username-updates/
 audit_date: '2018-09-24'
-title: MyRackspace username updates
+title: MyRackspace Login and Username Updates
```
[the remark this answers is not in the document]
Comment (butlercp) [reply content/dedicated-hosting/myrackspace-username-updates.md]: Will this change the URL of the KB?  I just noticed what I think is that the title is used in the URL creation for the article.

[the remark this answers is not in the document]
Comment (catlook) [reply content/dedicated-hosting/myrackspace-username-updates.md]: no...the url comes from the permalink.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-2ba59b9e7e/how-to/).

**Comment by catlook** [comment]
Merge on 10/18 - no sooner.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-2bb57b8d8f/how-to/). :bow:

**Pull request merged**